io/disk/fiotest.py.data/fio_rand_RW.yaml

```diff
@@ -0,0 +1,9 @@
+fio_randRW.yaml:
+parameters:
+    disk: '/dev/sdd'
```

Comment:
I would prefer to have value of `disk` empty, since when run unintentionally, test should do no harm to the disk.